nbuild/stdenv/install.py

```diff
@@ -20,6 +21,45 @@ def make_symlink(src, dst):
     os.symlink(src, dst)
 
 
+def make_cp(*src, dest, root='', args=''):
+    package = current_build().current_package
+    for filename in src:
+        path = f'{package.install_dir}/{root}/{filename}/'
```

Comment:
`path` shouldn't end with a `/`, as it is a path to a file, and not a directory.
This also applies to `make_rm` `make_mv` `make_chmod` and `make_sed`
```suggestion
        path = f'{package.install_dir}/{root}/{filename}'
```